client/src/modules/roles/modal/roleActions.html

```diff
@@ -0,0 +1,33 @@
+<div class="modal-header">
+    <ol class="headercrumb">
+      <li class="static" translate>FORM.BUTTONS.ACTIONS</li>
+      <li class="title">
+        <span translate>{{RoleActionsCtrl.role.label}}</span>
+      </li>
+    </ol>
+</div>
+<div class="modal-body">
+
+  <div class="row">
+    <div class="col-lg-12">
+      <ul class="list-group">
+        <li  class="list-group-item" ng-repeat="action in RoleActionsCtrl.actions">
+          <label class="radio-inline">
+            <input type='checkbox' ng-model='action.affected' ng-true-value="1" ng-false-value="0" />
+            <span id="{{action.id}}" translate>{{action.description}}</span>
+          </label>
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
+
+<div class="modal-footer">
+  <div class=" pull-right">
+  <button type="button" class='btn inverse' ng-click='RoleActionsCtrl.close()' translate>FORM.BUTTONS.CLOSE</button>
```

Comment:
I don't think the `btn inverse` class exists.  Your choices are in the [bootstrap documentation](https://getbootstrap.com/docs/3.3/css/#buttons).  I think we've been using `btn btn-default` for all our "cancel" buttons.